guides/doc-Release_Notes/topics/foreman-3.9.0.adoc

```diff
@@ -0,0 +1,159 @@
+= Foreman 3.9.0
+
+A full list of changes is available on https://projects.theforeman.org/issues?set_filter=1&sort=id%3Adesc&status_id=closed&f%5B%5D=cf_12&op%5Bcf_12%5D=%3D&v%5Bcf_12%5D%5B%5D=1765[Redmine]
+
+== Headline Features
+
+=== New All Hosts page features
+
+The All Hosts page has been revamped to include features like Host Create and Register buttons, bulk host deletion, and faster links to each host. Users will now have the ability to do many CRUD tasks from the All Hosts page instead of having to go to other pages.
```

Comment:
I think this is about the new experimental All Hosts page. AFAIK the existing All Hosts already has these.

cc @jeremylenz

Reply:
Yes, this is about the new page.

Reply:
```suggestion
=== New All Hosts page features

The new All Hosts page has been revamped to include features like Host Create and Register buttons, bulk host deletion, and faster links to each host. Users will now have the ability to do many CRUD tasks from the new All Hosts page instead of having to go to other pages.
```

Reply:
Isn't this the first version to even introduce the new hosts page? Shouldn't it explain that it was introduced as a new experimental feature, intended to replace the existing page?

Reply:
Whatever works for me. @jeremylenz let me know what you'd like to do.